### PR TITLE
Tso ha

### PR DESCRIPTION
**Pull request by weigangchen1**

### CODE REVIEW

File: examples/k8s/value_tso_HA.yaml [hunk 1]
```diff
@@ -0,0 +1,449 @@
...
+  image: byconity/byconity:0.1.0-GA
+  imagePullPolicy: IfNotPresent
+  hdfs_address: hdfs://byconity-hdfs-namenodes:8020 # can using your own hdfs
+  #  use_existing_fdb: true
+  #fdb_cluster_file: byconity_fdb:Is0hBgl6iICdHuspBmhAODmD5WISXKzI@192.168.224.150:4501,192.168.226.83:4501,192.168.228.152:4501
+  ports:
+    tcp: 9000
+    http: 8123
+    rpc: 8124
+    tcpSecure: 9100
```
Comment (dmthuc) [inline examples/k8s/value_tso_HA.yaml:18]: we do not need tcp, tcpSecure, https, exchange, exchangeStatus port for TSO

File: examples/k8s/value_tso_HA.yaml [hunk 2]
```diff
@@ -0,0 +1,449 @@
...
+              hostname: byconity-tso-1.byconity-tso.byconity-test.svc.cluster.local
+              port: 9445
+          - server:
+              id: 2
+              hostname: byconity-tso-2.byconity-tso.byconity-test.svc.cluster.local
+              port: 9445
+
+
+  daemonManager:
+    replicas: 1 # Please keep single instance now, daemon manager HA is WIP
```
Comment (dmthuc) [inline examples/k8s/value_tso_HA.yaml:179]: DM no need HA,

File: examples/k8s/value_tso_HA.yaml [hunk 3]
```diff
@@ -0,0 +1,449 @@
...
+      rpc.client.timeout: 20000
+      rpc.client.write.timeout: 20000
+
+  tso:
+    replicas: 3
+    image: ""
+    podAnnotations: { }
+    resources:
+      limits:
+        cpu: "4"
```
Comment (dmthuc) [inline examples/k8s/value_tso_HA.yaml:107]: seems the request for cpu is too high, maybe 3 is enough

File: examples/k8s/value_tso_HA.yaml [hunk 4]
```diff
@@ -0,0 +1,449 @@
...
+    tls: [ ]
+    #  - secretName: byconity-example-tls
+    #    hosts:
+    #      - byconity-example.local
+
+# For more detailed usage, please check fdb-kubernetes-operator API doc: https://github.com/FoundationDB/fdb-kubernetes-operator/blob/main/docs/cluster_spec.md
+fdb:
+  enabled: true
+  enableCliPod: true
+  version: 7.1.15
```
Comment (dmthuc) [inline examples/k8s/value_tso_HA.yaml:326]: let change it to 7.1.27

File: examples/k8s/value_tso_HA.yaml (hunk 1 above)
Comment (weigangchen1) [reply examples/k8s/value_tso_HA.yaml:18]: this is not for tso only. it's for all components

File: examples/k8s/value_tso_HA.yaml (hunk 4 above)
Comment (weigangchen1) [reply examples/k8s/value_tso_HA.yaml:320]: should i need to test it before release? i'm still using 7.1.15

File: examples/k8s/value_tso_HA.yaml (hunk 3 above)
Comment (weigangchen1) [reply examples/k8s/value_tso_HA.yaml]: i remove this part.

**Review by dmthuc — APPROVED** [review]

**Pull request merged**